src/doc/docs/decomposition.md

```diff
@@ -244,9 +243,8 @@ public class MyFeatureBundle implements GuiceyBundle {
 As bundle has almost complete access to configuration, it can use [disables](guide/disables.md)
 to substitute application functions.
 
-For example, it is known that application use `ServiceX` (from some core module in organization),
-but this bundle requires modified service. It can disable core module, installing feature 
-and register customized module:
+For example, it is known that an application uses `ServiceX` from some core module provided by the organization. Your module 
+requires a modified service. Your bundle may disable the core module, and install a replacement customized module:
```

Comment:
"and install a replacement customized module" -> I think its not clear what is replacement and would be better like this: "and install a customized module as a replacement"